Extensions/xkit_preferences.js

```diff
@@ -262,7 +262,7 @@ XKit.extensions.xkit_preferences = new Object({
 				}
 
 				// This is awful but at least it works.
-				var my_version = parseFloat(XKit.tools.replace_all(XKit.version, "\\.", ""));
+				var my_version = parseFloat(XKit.tools.replace_all(XKit.version.toString(), "\\.", ""));
```

Comment:
unnecessary; `XKit.version` is always a string since it's major.minor.patch format